packages/type-merging-directives/tests/typeMergingDirectivesTransformer.test.ts

```diff
@@ -0,0 +1,313 @@
+import { print } from 'graphql';
+
+import { makeExecutableSchema } from '@graphql-tools/schema';
+import { parseSelectionSet } from '@graphql-tools/utils';
+
+import { typeMergingDirectives } from '../src';
+
+describe('type merging directives', () => {
+  const { typeMergingDirectivesTypeDefs, typeMergingDirectivesTransformer } = typeMergingDirectives();
+
+  test('adds base selection sets', () => {
+    const typeDefs = `
+      ${typeMergingDirectivesTypeDefs}
+      scalar _Key
+
+      type Query {
+        _user(key: _Key): User @merge
+      }
+
+      type User @base(selectionSet: "{ id }") {
```

Comment:
What happens if there’s a directive and static config? Do the definitions merge or override?